Gems/Atom/Asset/ImageProcessingAtom/Assets/Config/GSI.preset

```diff
@@ -0,0 +1,56 @@
+{
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "MultiplatformPresetSettings",
+    "ClassData": {
+        "DefaultPreset": {
+            "UUID": "{C5E76E09-39FA-411F-B2E2-15B47BB6AB5F}",
+            "Name": "GSI",
+            "OutputTypeHandling": "UseInputFormatAndBitDepth",
```

Comment:
The UseInputFormatAndBitDepth name seems redundant since the implementation actually is using exact format of source image. 
I think "UseInputFormat" or "UseSourceFormat" is a better name.

Reply:
Sounds good, I will update with `UseInputFormat`